src/thunderbots/CMakeLists.txt

```diff
@@ -45,89 +46,79 @@ catkin_package(
 include_directories(
         ${catkin_INCLUDE_DIRS}
         ${PROTOBUF_INCLUDE_DIRS}
-        # We don't use the "include" directory, so specify the current directory to capture all the files
-        ./
-)
+        ${CMAKE_CURRENT_SOURCE_DIR}
+        )
 
+# Find all the generated protobuf source files
 file(GLOB PROTO_SRCS "${CMAKE_CURRENT_SOURCE_DIR}/proto/*.pb.cc")
 
+file(GLOB_RECURSE BACKEND_INPUT_SRC LIST_DIRECTORIES false CONFIGURE_DEPENDS
+        ${CMAKE_CURRENT_SOURCE_DIR}/backend_input/*.cpp
+        ${CMAKE_CURRENT_SOURCE_DIR}/geom/*.cpp
+        )
```

Comment:
I think we also want to GLOB our headers here so that CLion can use intellisense on them. Pretty sure that's what we're doing in our current implementation.

Reply:
Yes you're right. Will fix.